src/gsi/crtm_interface.f90

```diff
@@ -1730,6 +1761,7 @@ subroutine call_crtm(obstype,obstime,data_s,nchanl,nreal,ich, &
             ges_qsat (ixp,iy ,k, itsigp)*w10+ &
             ges_qsat (ix ,iyp,k, itsigp)*w01+ &
             ges_qsat (ixp,iyp,k, itsigp)*w11)*dtsigp
+     rh(k) = q(k)/qsat(k)  !added for cloud fraction computation
```

Comment:
@azadeh-gh The variable `qsat` is declared but not defined.

Reply:
I think `qsat` should `qs`

Reply:
@emilyhcliu  the qsat is defined in line 1152